composer.json

```diff
@@ -87,7 +90,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.3.x-dev"
+            "dev-feature/symfony3-compatibility": "2.3.x-dev"
```

Comment:
I think we do not need any branch alias for 2.x branches. A branch alias is required only on the master branch to match the next major release (3.0 ATM).